draft-ietf-quic-http.md

```diff
@@ -583,12 +673,34 @@ server to client, as in HTTP/2.  It defines no flags.
 
 The payload consists of:
 
-  Promised Stream ID:
-  : A 32-bit Stream ID indicating the QUIC stream on which the response will be
-    sent
+Push ID:
+: A 32-bit identifier for the server push request.  A push ID is used in push
```

Comment:
It really sounds odd to say "server push request". It's not a request... how about "promised resource"?